samples/Gaussian_initial_state/Program.qs

```diff
@@ -0,0 +1,24 @@
+namespace Gaussian_initial_state {
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic; 
+    open Microsoft.Quantum.Arrays;
+
+    @EntryPoint()
+    operation RunProgram() : Unit {
+        let N = 7;
```

Comment:
Could we add some docstrings here to explain what these variables are and how you decided on the values? (e.g. why is the standard deviation 2**N/6?)